src/plugins/Gitcoin/PreviewCard.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { makeStyles, createStyles, Theme, Typography } from '@material-ui/core'
+const border = '1.5px solid rgb(0, 154, 87)'
+interface PreviewCardProps {
+    title: string
+    image: React.ReactNode
+    line1: string
```

Comment:
```suggestion
    lines: [string, string, string, string]
```
use array maybe better?

Reply:
The meaning of it might change when it is used with logic so no need to change. And use non primitive props may cause a rerender everytime.